source/Calamari/Calamari.csproj

```diff
@@ -20,10 +20,10 @@
     <ApplicationManifest>Calamari.exe.manifest</ApplicationManifest>
   </PropertyGroup>
   <PropertyGroup Condition="!$([MSBuild]::IsOSUnixLike())">
-    <TargetFrameworks>net40;net452;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>net40;net452;net6.0</TargetFrameworks>
   </PropertyGroup>
   <PropertyGroup Condition="$([MSBuild]::IsOSUnixLike())">
-    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(TargetFramework)' == 'net40' OR '$(TargetFramework)' == 'net452' ">
     <DefineConstants>$(DefineConstants);IIS_SUPPORT;WINDOWS_CERTIFICATE_STORE_SUPPORT</DefineConstants>
```

Comment:
I'm surprised that there don't seem to be any constants defined for `net6.0` (unless there's something in the base msbuild files that automatically comes with the framework?).

Are there any parts of the Calamari project where we're switching based on constants other than the above two (`IIS_SUPPORT`, `WINDOWS_CERTIFICATE_STORE_SUPPORT`)?